landing-page/scripts/paco_api.py

```
@@ -88,10 +106,27 @@ def classify_stafflines(
         )
         resp = conn.getresponse()
         raw = resp.read()
+    except socket.timeout as exc:
```

Ruff (0.16.1)

[warning] Replace aliased errors with `TimeoutError`

Replace `socket.timeout` with builtin `TimeoutError`

(UP041)